src/styl/_components/_breakingnews.styl

```diff
@@ -6,60 +6,53 @@
 // Markup: breakingnews.twig
 //
 // Styleguide: Components.BreakingNews
-/:root
-/[data-theme=light]
-/[data-theme] [data-theme=light]
-    --breakingnews-title-color: var(--color-highlight)
-
-/[data-theme=dark]
-/[data-theme] [data-theme=dark]
-    --breakingnews-title-color: var(--color-highlight-foreground)
 
 .breakingnews
+    _p(1 0)
     _fontload($fontstack-secondary)
-    display flex
-    background var(--color-bg-primary)
+    background var(--color-highlight)
+    color var(--color-highlight-foreground)
     font-weight bold
-    flex-wrap nowrap
-    align-items stretch
-    justify-content stretch
     border 0 // no border even it’s box
 
-    &-label
-    &-title
-    &-cta
+    &-content
```

Comment:
What's the point of having 2 containers?

Reply:
in use we need to have the breaking news background who takes all screen width and the content take the size of page-content.

Reply:
I don't see it in your code.. Try to build your component on its own, without thinking of its context

Reply:
thanks, for your advice